tools/autograd/gen_python_functions.py

```diff
@@ -591,7 +591,10 @@ def get_python_binding_arguments(declaration):
                 # produce a compile-time error that is obvious
                 has_tensor_return = True
 
-        is_like_function = name.endswith('_like')
+        # The distinguishing characteristic of "like" functions is that they take
+        # a tensor and use it to compute some parameters of the new tensor,
+        # but they are still fundamentally factory functions
+        is_like_function = name.endswith('_like') or name.startswith('new_')
```

Comment:
just rename the variable to is_like_or_new_function ?

Reply:
I did... but then I have to rename more variables XD

Reply:
(I'll renaming)